admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/controller/DeviceControllerTest.java

```diff
@@ -415,7 +415,7 @@ public void t011createDeviceFailTest() throws Exception {
 		
 		MasterDataTest.checkResponse(mockMvc.perform(MockMvcRequestBuilders.post("/devices")
 				.contentType(MediaType.APPLICATION_JSON).content(mapper.writeValueAsString(deviceDtoReq))).andReturn(),
-				"KER-MSD-339");
```

Comment:
why for failed test cases we are adding null or modify the test name

Reply:
Modified the name sir.